### ISSUE TITLE
menu ropbin is ready crash on n3ds eur

### ISSUE THREAD

**Issue by zewr** [yellows8/3ds_smashbroshax#10]
Hi!

I'm on New 3DS and I have the hax working fine with release version. It triggers in like two seconds. But the 3DS crashes to the Home Menu when trying to launch hblauncher. The screen reads "menu ropbin is ready" just before the crash.

I think the problem might be that my n3ds is softbricked. I failed to sysUpdate it from 9.9 to 9.2 previously using browserhax, and now my system is a hybrid between both firmwares with no access to the browser (which is why I'm trying SmashHax). So I tried both otherapp payloads (for 9.2 and 9.9) and none of them work.

Question is, is that the real issue, or might this error be caused by something else?

I'm using a non-updated original version of Smash 3DS EUR and its 1.0.0 pcap. It asks for updates on boot but starts just fine when no need to save profile is selected.

Thank you for your help!

**Comment by yellows8** [comment]
What payload did you select?

**Comment by zewr** [comment]
I tried both New 9.9.0.26E from otherapp selector (original native firm of my n3ds) and New9.2.0.20E, which is the one I was trying to sysupdate to when the process failed and I got a softbrick. Both renamed to smashpayload.bin at the root  of the sd.

 Both fail at the same point... Do you think the problem is caused because of that?

Thank you!

**Comment by yellows8** [comment]
If that affected more than just the browser that may be an issue.

**Comment by zewr** [comment]
Ok, one more thing, I read a topic here about US cart version of SB shipping on v1.0.1. How can I check which version is the european one? That might be an issue too and I can't find it on the internet or inside the game menu... Sorry to bother you with this.

**Comment by yellows8** [comment]
The version is displayed on the top-screen at the title-screen(for >v1.0.0 at least).

**Comment by zewr** [comment]
Yeah, Ive seen it on a different 3DS with an updated version of Smash. Then I guess it's version 1.0.0, I was using the correct pcap, and the problem is actually that the payload doesn't work for my softbricked hybrid of a n3ds. 

Thank you for your time anyway!

**Comment by zewr** [comment]
To anyone interested: I finally made it work. After many tries, Smash Hax launched the hbl in my softbricked New3DS, and I could install MenuHax and downgrade from there.
I had to:
- Delete all related data in my SD card
- Use 9.2.0.20E otherapp payload (my native_firm was actually already downgraded)
- Reset my cartridge (boot pressing A+B+X+Y), hence lose all my savegames.

I have no explanation as to why that last point had any effect, actually, it may have been totally coincidentally. But the fact is, as soon as I did it, it worked.

Thanks for your help anyway @yellows8 !!
